### PR TITLE
Merge default and four token modules

### PR DESCRIPTION
**Pull request by rsaestrela**
This commit merges the token module into a single client that can be used for default and four account structures. It also adds `token_format` to `TokenResponse`.

### CODE REVIEW

File: src/main/java/com/checkout/tokens/CardTokenResponse.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package com.checkout.tokens;
```
Comment (Armando-Ibarra) [inline src/main/java/com/checkout/tokens/CardTokenResponse.java:1]: There is a missing property called `name`.

File: src/main/java/com/checkout/tokens/CardTokenRequest.java [hunk 2]
```diff
@@ -12,15 +12,13 @@
 @AllArgsConstructor
 public final class CardTokenRequest {
 
-    private final String type = "card";
+    private final TokenType type = TokenType.CARD;
 
     private String number;
 
-    @SerializedName("expiry_month")
```
Comment (Armando-Ibarra) [inline src/main/java/com/checkout/tokens/CardTokenRequest.java:19]: Please include the Serialized name.

**Review by Armando-Ibarra — CHANGES_REQUESTED** [review]
Nice work.
Just minor comments.

File: src/main/java/com/checkout/tokens/CardTokenResponse.java (hunk 1 above)
Comment (rsaestrela) [reply src/main/java/com/checkout/tokens/CardTokenResponse.java:1]: Nice catch!

**Pull request merged**